modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/block/processing/BlockAcceptanceManager.scala

```diff
@@ -37,7 +37,8 @@ object BlockAcceptanceManager {
 
       def acceptBlocksIteratively(
         blocks: List[Signed[Block]],
-        context: BlockAcceptanceContext[F]
+        context: BlockAcceptanceContext[F],
+        maybeOrdinal: Option[SnapshotOrdinal]
```

Comment:
when you accept the block, you should already know the snapshot the block belongs to 🤔 unless it's used on L1 where you don't know the snapshot but you know the latest one